test/examples/change_streams.js

```diff
@@ -101,9 +101,9 @@ describe('examples(change-stream):', function() {
       const collection = db.collection('inventory');
       const changeStream = collection.watch();
 
-      let resumeToken, newChangeStream;
+      let newChangeStream;
       changeStream.on('change', next => {
-        resumeToken = next._id;
+        const resumeAfter = next._id;
```

Comment:
FTR, this would be functionally the same as `const resumeAfter = changeStream.resumeToken`, which I believe is part of our public api (or is at least supposed to be).